interface/login/login.php

```diff
@@ -341,11 +341,22 @@ function transmit_form(element) {
                         <a href="../../acknowledge_license_cert.html" target="main"><?php echo xlt('Acknowledgments, Licensing and Certification'); ?></a>
                     </p>
                 </div>
-                <div class="product-registration-modal" style="display: none">
+                <div class="product-registration-modal modal fade">
+                <div class="modal-dialog modal-dialog-centered">
+                <div class="modal-content">
+                <div class="modal-header"></div>
+                <div class="modal-body">
                     <p class="context"><?php echo xlt("Register your installation with OEMR to receive important notifications, such as security fixes and new release announcements."); ?></p>
                     <input placeholder="<?php echo xlt('email'); ?>" type="email" class="email w-100 text-body" />
                     <p class="message font-italic"></p>
                 </div>
+                <div class="modal-footer">
+                    <button class="btn btn-primary submit">Submit</button>
+                    <button class="btn btn-danger noThanks">No Thanks</button>
```

Comment:
I think those buttons should be using .btn-group to the wrapper.
Follow this comment https://github.com/openemr/openemr/issues/3160#issuecomment-600342688